### PR TITLE
Error check

### PR DESCRIPTION
**Pull request by hectorhuertas**
Simple approach to detecting errors. It only informs about them instead of exiting, to avoid leaving the graylog pod unready.

Not in love with this solution, but didn't want to add complexity. Happy to hear suggestions on better approaches.

### CODE REVIEW

File: graylog-configurer [hunk 1]
```diff
@@ -18,18 +20,18 @@ set_input()
 
   input_id=$(curl -s -XGET "${graylog_api}/system/inputs" | jq -r '.inputs[] | select(.title == "'"${input_name}"'") | .id')
   if [ ! "${input_id}" ]; then
-    curl -s -X POST -H "Content-Type: application/json" -d "${input_data}" "${graylog_api}/system/inputs"
+    curl -sf -X POST -H "Content-Type: application/json" -d "${input_data}" "${graylog_api}/system/inputs" || errors="${errors}failed to create ${input_name} input, "
```
Comment (ffilippopoulos) [inline graylog-configurer:23]:
```
errors=${errors}:"failed to create ${input_name} input, "
```
no reason for that comment, it is just how I would expect to see an append.

File: graylog-configurer [hunk 2]
```diff
@@ -89,10 +92,15 @@ if [ "${ELASTICSEARCH_CUSTOM_TEMPLATE}" ]; then
...
   custom_template=$(echo "${ELASTICSEARCH_CUSTOM_TEMPLATE}" | jq -c)
-  curl -s -X PUT -H "Content-Type: application/json" -d "${custom_template}" "http://${ELASTICSEARCH_AUTHORITY}/_template/graylog-custom-template"
+  curl -sf -X PUT -H "Content-Type: application/json" -d "${custom_template}" "http://${ELASTICSEARCH_AUTHORITY}/_template/graylog-custom-template" || errors="${errors}failed to update elasticsearch custom template, "
   sleep 5
   # Rotate the active index to activate the new template
-  curl -s -X POST "${graylog_api}/system/deflector/cycle"
+  curl -sf -X POST "${graylog_api}/system/deflector/cycle" || errors="${errors}failed to cycle active index, "
+fi
+
+if [ "${errors}" ]; then
```
Comment (ffilippopoulos) [inline graylog-configurer:101]:
```
if [[ !  -z  $errors  ]]; then
```
but I guess again just personal preference when reading a bash script

**Review by ffilippopoulos — APPROVED** [review]

File: graylog-configurer (hunk 1 above)
Comment (hectorhuertas) [reply graylog-configurer:23]: The script runs in ash (default alpine shell). Still, I haven't seen that syntax before, do you have a documentation source? Does the colon has some special meaning?

File: graylog-configurer (hunk 2 above)
Comment (hectorhuertas) [reply graylog-configurer:101]: Same as before, not a bash script. Next iteration will be in go though

File: graylog-configurer (hunk 1 above)
Comment (ffilippopoulos) [reply graylog-configurer:23]: you are right, confusing it for bash

**Review by george-angel — APPROVED** [review]

**Pull request merged**